README.md

```diff
@@ -0,0 +1,54 @@
+broadway/coding-standard
+========================
+
+The coding standard of the Broadway project.
+
+# Installation
+
+```
+$ composer require --dev broadway/coding-standard
+```
+
+# Configuration
+
+## PHP CS fixer
+
+1. create a `.php_cs` file referencing the `.php_cs.dist` template:
+
+```php
+# .php_cs
+
+<?php
+
+$config = require 'vendor/broadway/coding-standard/.php_cs.dist';
+
+$config->setFinder(
+    \PhpCsFixer\Finder::create()
+        ->in([
+            getcwd() . '/src',
+            getcwd() . '/test',
+        ])
+);
+
+return $config;
+
+```
+
+2. add `.php_cs.cache` to your `.gitignore`.
```

Comment:
🐛 `s/cache/dist/`?

Reply:
PHP CS fixer creates a .php_cs.cache file which you don't need in your repo.